content/home/card_10/en-US.md

```diff
@@ -1,11 +1,11 @@
 ---
 # card 10
-title: "Protect yourself with Firefox Monitor"
-image: "monitor.jpg"
-link_url: "https://monitor.firefox.com/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=card"
-tag_label: "Security"
-size: "small"
-aspect_ratio: "16-9"
-include_highres_image: true
-desc: "Find out what hackers already know about you and get alerts when data breaches put you at risk."
+title: "The human costs of content moderation"
+image: "irl.png"
+link_url: "https://irlpodcast.org/season4/episode4/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=card"
+tag_label: "Podcast"
+size: "medium"
```

Comment:
The size of these cards changes from `small` to `medium`, so I'll have to update the home page HTML in bedrock to adjust to this new layout.

This does pose a new problem perhaps though: once this is fixed on dev, how best do we coordinate a prod push along with a card update, without showing a temporarily broken home page?

Reply:
It does pose a problem. Let's talk about it. I think we'll need to do something with a switch and separate cards or even a new template.